lib/conode/consts.go

```diff
@@ -1,3 +1,3 @@
-package defs
+package conode
 
 var DefaultPort string = "2000"
```

Comment:
We should think about removing that i think. It should be embedded in the application using this lib.